### ISSUE TITLE
Twemoji is changing HTML within a JSON, causing escaping issues

### ISSUE THREAD

**Issue by guygintonspot** [twitter/twemoji#370]
I work with a service that adds widgets to webpages, in the form of HTML inside of JSON objects. Twemoji is changing emojis within this HTML inside of JSON and causing an escaping issue. Is there a way for the page to prevent Twemoji from affecting widgets on a page?

**Comment by WebReflection** [comment]
is such JSON inside a script tag on the page? twemoji itself doesn't change anything automatically, so it'd be weird to hear about this

**Comment by guygintonspot** [comment]
Yes. The script tag creates components, such as a comment section.

**Comment by WebReflection** [comment]
So, as you can see, script nodes are not parsed:
https://github.com/twitter/twemoji/blob/master/scripts/generate#L264

This means you are parsing strings instead of creating proper nodes, and I suggest you use a template tag to inject your content first, and then parse its nodes, and then append its content.

Otherwise you should provide the minimal exact situation that causes issues.

**Comment by guygintonspot** [comment]
Sorry, I edited my comment to correct my mistake. A script creates divs that are being changed.  I want to know if we can add something to the "shouldntBeParsed" nodes, such as the class of the parent div, to avoid these issues.
